### ISSUE TITLE
Accessibility Select - Can't navigate to select with keyboard if in initially closed panel

### ISSUE THREAD

**Issue by GitJacek** [primefaces/primeng#17545]
### Describe the bug

There is an issue with keyboard accessibility when Select is used inside initially collapsed Panel.
When Panels input collapsed initially is set to false keyboard accessibility works as exptected.

### Reason for not contributing a PR

- [ ] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/uvcatr26?file=src%2Fapp%2Fpanel-toggleable-demo.html,src%2Fapp%2Fpanel-toggleable-demo.ts

### Environment

Win11

### Angular version

19.0.1

### PrimeNG version

v19

### Steps to reproduce the behavior

1. Go to https://stackblitz.com/edit/uvcatr26?file=src%2Fapp%2Fpanel-toggleable-demo.html,src%2Fapp%2Fpanel-toggleable-demo.ts,package.json
2. Expand Panel using + icon.
3. Try to navigate to Select using 'Tab' key on keyboard.
4. Unable to navigate to Select.

If you change filterCollapsed value to false:

<img width="269" alt="Image" src="https://github.com/user-attachments/assets/0d5ed4f1-2a45-41a7-8f82-61373c48da39" />

you will be able to navigate to select using 'Tab' key on keyboard.

For some reason tabindex input of select is ignored in first scenario.

### Expected behavior

User should be able to navigate to Select using keyboard even if panel is initially closed.

**Comment by alexist** [comment]
Hi,
To complete this issue : 

- When panel is initialy close, tab will focus on element inside the closed panel (p-input-text for exemple) .

- If panel is initialy open, when you close then re-open the panel, focus don't work on p-select, because tabindex is still -1 :

![Image](https://github.com/user-attachments/assets/477240cd-efbe-4703-a059-0133ae958241)
